app/api/v2/models/user_model.py

```diff
@@ -0,0 +1,65 @@
+from manage import db
+import datetime
+
+
+class UserModel:
+    """ class that """
```

Comment:
This comment is incomplete.

Reply:
I have fixed this. Thanks for your positive criticism